src/AppBundle/Entity/Report/Traits/ReportAssetTrait.php

```diff
@@ -9,6 +9,19 @@
 
 trait ReportAssetTrait
 {
+    /**
+     * Titles matching this will be included in the count for "Cash" in summary page
+     * Note: it relies on the translation (see report-assets.en.yml form.choices) for historical reasons
+     *
+     */
+    private static $cashAssetTitles = [
```

Comment:
Wouldn't work with non-english ;)

Reply:
the db only stores titles, due to the fact that types were added later. Fixing that would need a big data migration that was never prioritised. I'd say we do that next time we work on assets, that never changed for over 1 year

Reply:
it won't work if we rename the translation. I'll add a note into the translation file itself as a comment, so that we won't forget to update the code as well